client/app/vps/modal/shortcut/reverse-dns/vps-reverse-dns.controller.js

```diff
@@ -46,8 +46,10 @@ class VpsReverseDnsCtrl {
                 if (data && data.state) {
                     switch (data.state) {
                     case "ERROR" :
-                        this.CloudMessage.error(this.$translate.instant("vps_configuration_reversedns_fail"))
-                        break;
+                        this.CloudMessage.error(this.$translate.instant("vps_configuration_reversedns_fail"));
+                        const messages = !_.isEmpty(data.messages) ? data.messages : [];
+                        _.forEach(messages, (message) => this.CloudMessage.error(message.message || message));
```

Comment:
ES6 version, messages. forEach((message) => this.CloudMessage.error(message.message || message)));